br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -504,8 +526,129 @@ func (em *ErrorManager) ResolveAllConflictKeys(
 	return errors.Trace(g.Wait())
 }
 
+// ReplaceConflictKeys query all conflicting rows (handle and their
+// values) from the current error report and resolve them concurrently
```

Comment:
```suggestion
// values) from the current error report and resolve them
```

We didn't resolve them concurrently now